Author: Robie Basak <robie.basak@oss.qualcomm.com>
Description: cherry-pick config yaml from upstream
 FastRPC 1.0.2 requires this file, but it was erroneously missed from the
 upstream release tarball. We can patch it in from the upstream source tree
 instead, and this can be dropped when we update to a later upstream release
 that does include it in the upstream release tarball.
Origin: upstream, https://github.com/linux-msm/hexagon-dsp-binaries/commit/86297647dfa8104c26037e59b9360998b9d6e3a6
Bug: https://github.com/linux-msm/hexagon-dsp-binaries/issues/45
Forwarded: non-needed
Last-Update: 2026-02-25

--- /dev/null
+++ b/00-hexagon-dsp-binaries.yaml
@@ -0,0 +1,36 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) Qualcomm Technologies, Inc. and/or its subsidiaries.
+#
+# Machine entries are sorted alphabetically by machine name (key)
+# This sorting order ensures maintainability and stability as new properties are added
+machines:
+  "Qualcomm SA8775P Ride":
+    DSP_LIBRARY_PATH: "sa8775p/Qualcomm/SA8775P-RIDE/dsp"
+  "Qualcomm SA8775P Ride Rev3":
+    DSP_LIBRARY_PATH: "sa8775p/Qualcomm/SA8775P-RIDE/dsp"
+  "Qualcomm Technologies, Inc. DB820c":
+    DSP_LIBRARY_PATH: "apq8096/Qualcomm/db820c/dsp"
+  "Qualcomm Technologies, Inc. Hamoa IoT EVK":
+    DSP_LIBRARY_PATH: "x1e80100/Qualcomm/Hamoa-IoT-EVK/dsp"
+  "Qualcomm Technologies, Inc. Kaanapali MTP":
+    DSP_LIBRARY_PATH: "kaanapali/Qualcomm/Kaanapali-MTP/dsp"
+  "Qualcomm Technologies, Inc. Lemans EVK":
+    DSP_LIBRARY_PATH: "sa8775p/Qualcomm/IQ9075-EVK/dsp"
+  "Qualcomm Technologies, Inc. Monaco EVK":
+    DSP_LIBRARY_PATH: "qcs8300/Qualcomm/IQ8275-EVK/dsp"
+  "Qualcomm Technologies, Inc. QCS615 Ride (IQ-615 Beta EVK)":
+    DSP_LIBRARY_PATH: "qcs615/Qualcomm/QCS615-RIDE/dsp"
+  "Qualcomm Technologies, Inc. QCS8300 Ride":
+    DSP_LIBRARY_PATH: "qcs8300/Qualcomm/QCS8300-RIDE/dsp"
+  "Qualcomm Technologies, Inc. QRB4210 RB2":
+    DSP_LIBRARY_PATH: "qrb4210/Thundercomm/RB2/dsp"
+  "Qualcomm Technologies, Inc. Robotics RB1":
+    DSP_LIBRARY_PATH: "qcm2290/Thundercomm/RB1/dsp"
+  "Qualcomm Technologies, Inc. Robotics RB3gen2":
+    DSP_LIBRARY_PATH: "qcm6490/Thundercomm/RB3gen2/dsp"
+  "Qualcomm Technologies, Inc. Robotics RB5":
+    DSP_LIBRARY_PATH: "sm8250/Thundercomm/RB5/dsp"
+  "Qualcomm Technologies, Inc. SM8750 MTP":
+    DSP_LIBRARY_PATH: "sm8750/Qualcomm/SM8750-MTP/dsp"    
+  "Thundercomm Dragonboard 845c":
+    DSP_LIBRARY_PATH: "sdm845/Thundercomm/db845c/dsp"
